packages/core/template/project/_markbind/layouts/default.md

```diff
@@ -0,0 +1,67 @@
+<head-bottom>
+  <link rel="stylesheet" href="{{baseUrl}}/stylesheets/main.css">
+</head-bottom>
+
+<header sticky>
+  <navbar type="dark">
+    <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand">ProjectEx</a>
+    <li><a href="{{baseUrl}}/UserGuide.html" class="nav-link">User Guide</a></li>
+    <li><a href="{{baseUrl}}/DeveloperGuide.html" class="nav-link">Developer Guide</a></li>
+    <li><a href="{{baseUrl}}/AboutUs.html" class="nav-link">About Us</a></li>
+    <li><a href="https://github.com/se-edu" target="_blank" class="nav-link"><md>:fab-github:</md></a>
+    </li>
+    <li slot="right">
+      <form class="navbar-form">
+        <searchbar :data="searchData" placeholder="Search" :on-hit="searchCallback" menu-align-right></searchbar>
+      </form>
+    </li>
+  </navbar>
+</header>
+
+<div id="flex-body">
+  <nav id="site-nav">
+    <div class="site-nav-top">
+      <div class="fw-bold mb-2" style="font-size: 1.25rem;">Site Map</div>
+    </div>
+    <div class="nav-component slim-scroll">
+      <site-nav>
+* [Home]({{ baseUrl }}/index.html)
+* [User Guide]({{ baseUrl }}/UserGuide.html) :expanded:
+  * [Quick Start]({{ baseUrl }}/QuickStart.html)
+  * [Features]({{ baseUrl }}/Features.html)
+  * [FAQ]({{ baseUrl }}/FAQ.html)
+* [Developer Guide]({{ baseUrl }}/DeveloperGuide.html) :expanded:
+  * [Setting Up]({{ baseUrl }}/SettingUp.html)
+  * [Design]({{ baseUrl }}/Design.html)
+  * [Implementation]({{ baseUrl }}/Implementation.html)
+  * Project Guides
+    * [Documentation]({{ baseUrl }}/Documentation.html)
```

Comment:
These files highlight under the User Guide section rather than the Developer guide section
<img width="1158" alt="Screenshot 2024-03-04 at 11 16 40 AM" src="https://github.com/MarkBind/markbind/assets/59471320/ecc1d479-61f6-4e3e-992d-4d52788e2833">